lib/screens/home/local_widget/province_card.dart

```diff
@@ -72,23 +72,22 @@ class ProvinceCard extends StatelessWidget {
           String commune = NumberHelper.toKhmer(district[index].commune) + 'ឃុំ';
           String village = NumberHelper.toKhmer(district[index].village) + 'ភូមិ';
           bool isKrong = district[index].type == 'KRONG';
+          void Function()? onTap = () {
+            Navigator.pushNamed(context, RouteConfig.DISTRICT, arguments: district[index]);
+          };
           return Column(
             children: [
               Divider(height: 0),
               (isKrong || isKhan)
                   ? ListTile(
                       title: Text((isKrong ? 'ក្រុង' : 'ខណ្ឌ') + (district[index].khmer ?? '')),
                       subtitle: Text('$sangkat និង $village'),
-                      onTap: () {
-                        Navigator.pushNamed(context, RouteConfig.DISTRICT);
-                      },
+                      onTap: onTap,
                     )
                   : ListTile(
```

Comment:
we should just update the title & subtitle here, so we don't have to write 2 widgets

eg. 
```dart
String title, subtitle;
if(isKrong || isKhan){title = "sth", subtitle = "sth"}
else {title = "1", subtitle = "2"}
```